index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta content='Linkbrary' property='og:title'/>
+  <meta content='//www.wjyweekly.netlify.app/' property='og:url'/>
+  <meta content='세상의 모든 정보를 쉽게 저장하고 관리해 보세요' property='og:description'/>
+  <meta content='./img/page.png' property='og:image'/>
+  <title>Linkbrary</title>
+  <link rel="stylesheet" href="css/indexStyle.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <link
+  href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&family=Poppins:wght@400;700&display=swap"
+  rel="stylesheet"
+>
+</head>
+<body>
+  <!-- 네이게이션 바 -->
+  <nav class="navigationBar">
+    <div class="nav-container">
+      <a class="btn-home" href="./index.html">Linkbrary</a> <a class="btn btn-login" href="./login.html">로그인</a>
+    </div>
+  </nav>  
```

Comment:
@wjy308 
```suggestion
<div class="gnb">
  <nav class="container navbar">
      <a class="logo">Linkbrary
      <a class="btn btn-cta">로그인
```


이런형태는 어떨런지요? 

우선, `nav` 요소는 [링크들을 열거하는 박스로 활용되기위해 사용됩니다](https://developer.mozilla.org/ko/docs/Web/HTML/Element/nav). 
반면, 지금 구현하신 방향을 보면, nav안에 UI배치를 위한 div를 두었죠. 순서가 바뀐것 같아요. 

또한 container도 조금더 재활용 가능한 방향으로 구현을 시도해보아도 좋을것 같습니다! 
분명 모든 페이지에서 최대 넓이를 지정하고, 콘텐츠를 중앙에 위치하도록 설정하는 요구사항이 있을텐데
이를 제공하는 `container` 클래스를 만들어 사용해보면 어떨까요? 

위의 예시에 적용해본다면 
```css
.container {
  width: 100%;
  max-width: 768px;  /* 모바일 최대넓이 설정 */
  padding: 0px 20px; /* 좌우 패딩값 설정 */
  margin: 0px auto; /* 뷰포트가 더 큰 경우, mx-auto로 중앙정렬 */
}

.navbar {
  display: flex;
  justify-contents: space-between;
  align-items: center
}
```

이 두 클래스를 합쳐서 활용하는식으로 해보면 좋을것 같아요